Changes for 7.1 (#1119)
owncloud/docs-ocis: modules/ROOT/pages/deployment/services/s-list/notifications.adoc

:envvar_name: NOTIFICATIONS_TRANSLATION_PATH
include::partial$deployment/services/translations.adoc[]

== Sending Grouped Emails

The {service_name} service can initiate sending emails based on events stored in the configured store that are grouped into a `daily` or `weekly` bucket. These groups contain events that get populated e.g. when the user configures daily or weekly email notifications in his personal settings in the web UI. The notification settings are only visible in the web UI when enabled via the enironment variable xref:{s-path}/frontend.adoc[FRONTEND_CONFIGURABLE_NOTIFICATIONS]. If enabled and a user does not define any of the named groups for notification events, no event is stored.

Grouped events are stored for the TTL defined in `OCIS_PERSISTENT_STORE_TTL`. This TTL can either be configured globally or individually for the notification service via the `NOTIFICATIONS_STORE_TTL` environment variable.

Grouped events that have passed the TTL are removed automatically without further notice or sending!

To initiate sending grouped emails like via a cron job, use the following command:

[source,bash]
----
ocis notifications send-email
----

> Note that the command *mandatory* requires at least one option which is `--daily` or `--weekly`. Note that both options can be used together.

Reviewer: ```suggestion
Note that at least one option, `--daily` or `--weekly`, must be provided. Both options can be used together.
```
